opus/application/apps/ui/views.py

```diff
@@ -105,12 +107,31 @@ def api_last_blog_update(request):
             lastupdate = fp.read().strip()
     except:
         try:
-            log.error('api_last_blog_update: Failed to read file "%s"',
+            log.error('api_notifications: Failed to read file "%s"',
                       settings.OPUS_LAST_BLOG_UPDATE_FILE)
         except:
-            log.error('api_last_blog_update: Failed to read file UNKNOWN')
+            log.error('api_notifications: Failed to read file UNKNOWN')
 
-    ret = json_response({'lastupdate': lastupdate})
+    notification = None
+    notification_modify = None
+    try:
+        with open(settings.OPUS_NOTIFICATION_FILE, 'r') as fp:
+            notification = fp.read().strip()
+            try:
+                notification_modify = os.path.getmtime(settings.OPUS_NOTIFICATION_FILE)
+            except:
+                log.error('api_notification: Failed to read the modify date of file "%s"',
+                          settings.OPUS_NOTIFICATION_FILE)
+    except:
+        try:
+            log.error('api_notifications: Failed to read file "%s"',
```

Comment:
I think we don't want this to be an error. The blog update date should always be there, but it's perfectly normal for there to be no notification file. We don't want that condition logged as an error in the logs. Perhaps make it debug level instead (here and below).